### DISCUSSION TITLE
Scripting in Tasmota12.4.0.1 not functional

### DISCUSSION THREAD

**Discussion by HofeBY** [arendst/Tasmota#18081] [Q&A] [answered]
I downloaded the new Tasmota12.4.0.1 vom arendst and did a new Compiler run under Windows10 &
"MS Visual Studio  Code", like te TM version before.

Using my configuration before in user_config_override.h

Compiling without errors.
Then I flash the firmware.bin to my AZDelivery ESP32-DevKitCv4 also like before.
After configuration the Device like before, I notice that the Scriping is not funktional.

At the Webinterface of Tasota Device, >M Section deliver PowerCounter to Webinterface, like before.

But, no data ale published to IOBroker !
 >W and >J Section (to internal Webinterface) publish nothing to IOBroker.
What Do I have to change, to get it run like previos Version of Tasmota ?

**Comment by barbudor** [comment, marked as answer]
You don't tell what you have exactly written into your user_config_override.h
`>W` and `>J` are both optional features that requires specific compile flags

![image](https://user-images.githubusercontent.com/2996042/222221738-7beb9325-1418-4631-b130-5da3ccecbfdb.png)
![image](https://user-images.githubusercontent.com/2996042/222221828-7077e709-8fae-4e56-bb8d-74c133201a0a.png)

**Reply by HofeBY** [reply]
Thank you, barbudor
I add

 #undef USE_SCRIPT_WEB_DISPLAY
#define USE_SCRIPT_WEB_DISPLAY
 in user_config_override.h an did a neu flash to ESP32.

now it´s functional.
since TM12.4.0.1 this is needed to be added.
